File: Android/aryanmishra29_AryanMishra_2125csme1029_2/Patterns/Pattern_12.java
import java.util.*;

public class Pattern_12 {
    public static int fibo(int n){
        if(n==0 || n==1) return n;
        return fibo(n-1)+fibo(n-2);
    }

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n=scn.nextInt();
        int count=0;
        for(int i=0;i<n;i++){
            for(int j=0;j<=i;j++) System.out.print(fibo(count++)+"\t");
            System.out.println();
        }
        // write ur code here

    }
}

File: IOT/ritikgupta88_RitikGupta_2125csit1163_2/Patterns/Pattern12.cpp
#include <iostream>
using namespace std;
int fibo(int n);
int main(int argc, char **argv){
    int n,k=1;
    cin >> n;
    for(int i=0;i<n;i++){
        for(int j=0;j<=i;j++){
            if(i+j<=2*i){
            cout<<fibo(k)<<"\t";
            k++;
            }
        }
        cout<<endl;
    }
}
    int fibo(int n){
        if(n==1 )
        return 0;
        if(n==2)
        return 1;
       
        else 
        return(fibo(n-1)+fibo(n-2));
    
   //write your code here
   
}